src/table/interfaces.tsx

```diff
@@ -324,8 +326,19 @@ export interface TableProps<T = any> extends BaseComponentProps {
   /**
    * Use this property to activate advanced keyboard navigation and focusing behaviors.
    * When set to `true`, table cells become navigable with arrow keys, and the entire table has a single tab stop.
+   *
+   * By default, the keyboard navigation is active for tables with expandable rows.
    */
   enableKeyboardNavigation?: boolean;
+
+  /**
+   * Use this property to provide define expandable table rows. The expandableRows configuration object consists of
```

Comment:
Double verb: "provide define".

Shouldn't we add a colon at the end of the sentence?

Reply:
Thanks, updated 👍